blog/10-years-of-electron.md

```diff
@@ -0,0 +1,244 @@
+the project. Although source code commits are always the most visible contributions, we also have to
+acknowledge the effort of folks who report bugs, maintain userland modules, provide documentation
+and translations, and participate in the Electron community across cyberspace.
+Every contribution is invaluable to us as maintainers.
+
+**Before we continue with the rest of the blog post: thank you. ❤️**
+
+## How did we get here?
+
+**Atom Shell** was built as the backbone for GitHub’s [Atom editor](https://atom.io/), which launched in
+public beta in April 2014. It was built from the ground-up as an alternative to the web-based
+desktop frameworks available at the time (node-webkit and Chrome Embedded Framework). It had a
+killer feature: embedding Node.js and Chromium to provide a powerful desktop runtime for web
+technologies.
+
+Within a year, Atom Shell began seeing immense growth in capabilities and popularity.
+Large companies, startups, and individual developers alike had started building apps with it
+(some early adopters include [Slack](https://slack.com), [GitKraken](https://gitkraken.com), and
+[WebTorrent](https://webtorrent.io)), and the project was aptly renamed to **Electron**.
+
+From then on, Electron hit the ground running and never stopped. Here’s a look at our weekly
+download count over time, courtesy of [npmtrends.com](https://npmtrends.com/electron):
+
+![Electron weekly downloads graph over time](/assets/img/weekly-download-graph.png)
+
+Electron v1.0 was released in 2016, promising increased API stability and better docs and tooling.
+Electron v2.0 released in 2018 and introduced semantic versioning, making it easier for Electron
+developers to keep track of the release cycle.
+
+By Electron v6.0, we shifted to a regular 12-week major release cadence to match Chromium’s. This
+decision was a change in mentality for the project, bringing “having the most up-to-date Chromium
+version” from a nice-to-have to a priority. This has reduced the amount of tech debt between
+upgrades, making it easier for us to keep Electron updated and secure.
+
+Since then, we’ve been a well-oiled machine, releasing a new Electron version on the same day as
+every Chromium stable. By the time Chromium sped up their release schedule to 4 weeks in 2021,
+we were able to shrug our shoulders and increase our release cadence to 8 weeks accordingly.
+
+We’re now on Electron v23.0 (and counting), and are still dedicated to building the best runtime for
+building cross-platform desktop applications. Even with the boom in JavaScript developer tools in
+recent years, Electron has remained a stable, battle-tested stalwart of the desktop app framework
+landscape. Electron apps are ubiquitous nowadays: you can program with Visual Studio Code, design
+with Figma, communicate with Slack, and take notes with Notion (amongst many other use cases).
+We’re incredibly proud of this achievement and grateful to everyone who has made it possible.
```

Comment:
Potentially worth switching either vX _or_ vX.X.X in all instances, since that's generally how the ecosystem looks at versions.

Reply:
minor/patch probably doesn't matter here, e.g. "We're now on Electron v23" and "By Electron v6, we shifted to a regular 12-week major release cadence" flows better.

Reply:
[238eadf](https://github.com/electron/website/pull/382/commits/238eadfe656329205417f2d49e711ad1fb096378)